### DISCUSSION TITLE
Serenity framework in Mobile App

### DISCUSSION THREAD

**Discussion by ShifanilaSenthil** [serenity-is/Serenity#6857] [Help]
Hi, 
I need help getting my serenity project into a mobile app compatible with all resolutions. How could we do that?

**Comment by osmanaslancan** [comment]
Serenity enterprise has a application sample. https://demo.serenity.is

**Reply by ShifanilaSenthil** [reply]
Actually, it would be very helpful if there is a guidance in how to and where to start with for that mobile app
